airflow-core/tests/unit/core/test_configuration.py

```diff
@@ -1105,6 +1105,33 @@ def test_order_of_secrets_backends_and_kwargs_on_workers(
             for key, value in expected_backend_kwargs.items():
                 assert getattr(secrets_backend, key) == value
 
+    def test_default_validators(self):
```

Comment:
It'd be helpful to add a few more tests:
- Verify the exact order and names of the default validators, not just their count. This helps prevent accidental reordering.
- Consider injecting a dummy validator that raises an exception, then check that the exception bubbles up and that `is_validated` remains `False`.
- Run `validate` twice and ensure no additional changes occur. If there are migrations involved, demonstrate that re-running is safe.